src/components/Timeline/index.js

```diff
@@ -0,0 +1,167 @@
+import React from "react";
+import "./Timeline.css";
+import Icon from "../Icon/Icon";
+import H2 from "../H2";
+
+function Timeline() {
+  return (
+    <div className="timeline">
+      <H2 text="About Me." />
+      <div className="outer">
+        <div className="card">
+          <div className="info">
+            <div className="title__row">
```

Comment:
@AbdurrahmanSogbesan  can you notice the repeating code. Basically the card div is repeating with diff data. Thats how youll know your prop should be an array. So pass in an array of objects that has the values youre displaying in the timeline title, desceiption). You get ?